documentation/specs/Card.md

```diff
@@ -0,0 +1,247 @@
+# `Card` Component Specification
+
+## Overview
+
+A Card is a styled container that groups related content.
+
+### Use Cases
+
+- Relating a concise amount of information together
+- Ultimately very flexibile for a variety of use cases
+
+### Features
+
+- Multiple border style variants
+- Background options
+- Deconstructed parts for composable complex cards
+
+### Risks and Challenges
+
+- Ensuring the component can be composed to the variety of use cases while maintaining a simple API
+
+### Prior Art
+
+- [Polaris `<AlphaCard />`](https://polaris.shopify.com/components/layout-and-structure/alpha-card)
+- [Paste `<Card />`](https://paste.twilio.design/components/card)
+
+---
+
+## Design
+
+A `Card` is semantically straightforward. Essentially serving as a container with preset styles. The complexity is surfacing a sufficiently advanced API that can accommodate the use cases without becoming overburdened.
+
+Architecture proposed is to surface a basic `<Card />` component with `variety` and `background` props while exposing the deconstructed parts to allow for maximum composability for complex use cases. The underlying pieces will be the `<Card.Container />` which houses the `variant` prop for determining the overall style of the card, and then a `<Card.Area />` which contains the inner container properties such as padding and the `background`.
+
+### API
+
+```ts
+export type CardAs = "div" | "label" | "button" | "a" | "fieldset";
```

Comment:
**question**: is the idea to use the `UnstyledButton` to handle the `"button"` and `"a"` scenario?